test/powershell/Installer/WindowsInstaller.Tests.ps1

```diff
@@ -14,24 +14,22 @@ Describe "Windows Installer" -Tags "Scenario" {
         $global:PSDefaultParameterValues = $originalDefaultParameterValues
     }
 
-    Context "Universal C Runtime Download Link" {
-        $universalCRuntimeDownloadLink = 'https://www.microsoft.com/download/details.aspx?id=50410'
-        It "Wix file should have download link about Universal C runtime" -Pending {
-           (Get-Content $wixProductFile -Raw).Contains($universalCRuntimeDownloadLink) | Should Be $true
-        }
-        It "Should have download link about Universal C runtime that is reachable" {
-           (Invoke-WebRequest $universalCRuntimeDownloadLink.Replace("https://",'http://')) | Should Not Be $null
-        }
-    }
+    $universalCRuntimeDownloadLink = 'https://www.microsoft.com/download/details.aspx?id=50410'
+    $visualStudioCPlusPlusRedistributablesDownloadLink = 'https://www.microsoft.com/download/details.aspx?id=48145'
 
-    Context "Visual Studio C++ Redistributables Link" {
-        $visualStudioCPlusPlusRedistributablesDownloadLink = 'https://www.microsoft.com/download/details.aspx?id=48145'
-        It "WiX file should have documentation about Visual Studio C++ redistributables" -Pending {
-           (Get-Content $wixProductFile -Raw).Contains($visualStudioCPlusPlusRedistributablesDownloadLink) | Should Be $true
-        }
-        It "Should have download link about Universal C runtime that is reachable" {
-           (Invoke-WebRequest $visualStudioCPlusPlusRedistributablesDownloadLink.Replace("https://",'http://')) | Should Not Be $null
-        }
+    $downloadLinks = @(
+        @{ downloadLink = $universalCRuntimeDownloadLink; Name = 'Universal C Runtime' }
+        @{ downloadLink = $visualStudioCPlusPlusRedistributablesDownloadLink; Name = 'Visual Studio C++ 2015 Redistributables' }
+    )
+        
+    It "WiX (Windows Installer XML) file contains download link '<downloadLink>' for '<Name>'" -TestCases $downloadLinks -Test {
+        Param ([string]$downloadLink)
+        (Get-Content $wixProductFile -Raw).Contains($downloadLink) | Should Be $true
     }
 
-}
\ No newline at end of file
+    It "Download link '<downloadLink>' for '<Name>' is reachable" -TestCases $downloadLinks -Test {
+        Param ([string]$downloadLink)
+        (Invoke-WebRequest $universalCRuntimeDownloadLink.Replace("https://", 'http://')) | Should Not Be $null
```

Comment:
Why are you replacing HTTPS with HTTP?

Reply:
Also, perhaps if we just want to validate the download links haven't been removed, we should just check that you get a 200 status?

Reply:
The replacement of https with http was because I sometimes got errors, I can take it out if you don't like it.
If you take an invalid URL like e.g. `https://www.microsoft.com/download/details.aspx?id=5041000000`, then you still get a 200 response because I get redirected to an error page but the returned object is `$null`

Reply:
I think it's better to leave as HTTPS.  Depending on why it's failing, you could try it a few times before failing the test since this is `Scenario`.

For the $null vs 200 status, you should add a comment so other people understand why you made the choice.